fi/features.html

```diff
@@ -0,0 +1,458 @@
+
+    <div class="dropdown">
+        <h3>Tilastotiede</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/stats.html">Dokumentaatio</a></p>
+            <ul>
+              <li><strong>Satunnaismuuttujien tyypit:</strong>
+                <ul>
+                  <li><strong>Normaalijakaumat</strong></li>
+                  <li><strong>Tasajakaumat</strong></li>
+                  <li><strong>Bernoulli-jakaumat</strong></li>
+                  <li><strong>Binomijakaumat</strong></li>
+                  <li><strong>Hypergeometriset jakaumat</strong></li>
+                  <li><a href="http://docs.sympy.org/latest/modules/stats.html#random-variable-types">Lisää&hellip;</a>
+                </ul>
+              </li>
+              <li><strong>Todennäköisyys</strong></li>
+              <li><strong>Odotusarvo ja varianssi</strong></li>
+              <li><strong>Todennäköisyyden tiheysfunktio</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Kryptografia</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/crypto.html">Dokumentaatio</a></p>
+            <ul>
+              <li><strong>Siirtosalaus</strong></li>
+              <li><strong>Affiini salaus</strong></></li>
+              <li><strong>Bifid-salaus</strong></></li>
+              <li><strong>Vinegeren salaus</strong></></li>
+              <li><strong>Korvaussalaus</strong></></li>
+              <li><strong>Hillin salaus</strong></></li>
+              <li><strong>RSA</strong></></li>
+              <li><strong>Kid RSA</strong></></li>
+              <li><strong>Vaihtorekisterien lineaarinen palaute</strong></></li>
+              <li><strong>ElGamal-salaus</strong></></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Jäsennys</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/parsing.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Python-olioiden kääntäminen SymPy-olioiksi</strong></li>
+                <li><strong>Valinnainen implisiittinen moninkertaistuminen ja funktion sovellusjäsennys</strong></li>
+                <li><strong>Rajoitettu Mathematica ja Maxima - jäsennys</strong> <a href="http://live.sympy.org/?evaluate=from%20sympy.parsing.mathematica%20import%20mathematica%0A%23--%0Afrom%20sympy.parsing.maxima%20import%20parse_maxima%0A%23--%0Amathematica(%27Sin%5Bx%5D%27)%0A%23--%0Aparse_maxima(%27sqrt(x%5E2)%27)%0A%23--%0A">esimerkki SymPy-livessä</a></li>
+                <li><strong>Sovitetut jäsennysmuutokset</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Tulostaminen</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/printing.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Kaunis tulostaminen:</strong> ASCII/Unicode kaunis tulostaminen, LaTeX</li>
```

Comment:
"pretty printing" means "formatted output" and probably should be translated based on thin meaning rather than the English jargon word.